File: 1_iniciante/javascript/1009.js
/**
 * @source Beecrowd
 * @problem 1009 - Salary with Bonus
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2020-10-01
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let nome = lines.shift();
let salariofixo = parseFloat(lines.shift());
let vendames = parseFloat(lines.shift());

let vendbonus = (vendames * 15) / 100;
let final = vendbonus + salariofixo;

console.log("TOTAL = R$ " + final.toFixed(2));

File: 1_iniciante/c++/1009.cpp
/**
 * @source Beecrowd
 * @problem 1009 - Salary with Bonus
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2025-08-25
 */

#include <iostream>
#include <iomanip>

using namespace std;

int main()
{

    string nome;
    double salarioFixo, vendasEfetuadas;

    cin >> nome >> salarioFixo >> vendasEfetuadas;

    double salarioComBonus = salarioFixo + vendasEfetuadas * 0.15;
    cout << "TOTAL = R$ " << setprecision(2) << fixed << salarioComBonus << endl;

    return 0;
}